street_network/src/transform/classify_intersections.rs

```diff
@@ -34,8 +33,50 @@ fn guess_complexity(_inter: &Intersection, roads: Vec<OriginalRoad>) -> Intersec
         return Connection;
     }
 
-    // A MultiConnection is characterised by exactly two connected dividing lines (lines that
-    // separate traffic in different directions) and no traffic that crosses them.
+    // A MultiConnection is characterised by exactly one dividing line traveling through it (the
+    // line that separates traffic in different directions) and no traffic that crosses it.
+    if roads.len() == 3 {
+        let mut num_roads_in = 0;
+        let mut num_roads_out = 0;
+        let mut num_roads_inout = 0;
+        for or in roads {
```

Comment:
Code style: from @michaelkirk I picked up the good habit of trying to spell out longer names, unless the abbreviation was very standard or used frequently in the codebase. Could we try that here? Is this `outward_road`? (Which would be a weird name, because we determine below if it's even outward or not

Reply:
I have a habit of using short names for variables that are short lived over only a few lines, but I appreciate the push towards less obtuse naming. This case is bad no matter how you look at it though, I will change it to `road_id`.